### PR TITLE
Add models

### PR DESCRIPTION
**Pull request by wayzor9**

### CODE REVIEW

File: plants/models.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+from django.db import models
+from django.conf import settings
+
+User = settings.AUTH_USER_MODEL
+
+
+class Plant(models.Model):
+    plantId_id = models.IntegerField()
```
Comment (halypa) [inline plants/models.py]: Remove it

File: plants/models.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    name = models.CharField(max_length=200)
+    created = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return f"{self.name}, plantId: {self.plantId_id}"
+
+
+class ProfilePlant(models.Model):
+    user = models.ForeignKey("UserProfile", on_delete=models.CASCADE)
```
Comment (halypa) [inline plants/models.py]:
```
 class ModelName(TimeStampedModel):
     user = models.ForeignKey(USER, on_delete=models.CASCADE)
```
https://django-model-utils.readthedocs.io/en/latest/models.html#timestampedmodel

File: plants/models.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    name = models.CharField(max_length=200)
+    plant = models.ForeignKey(Plant, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.name
+
+
+class PlantPicture(models.Model):
+    image = models.ImageField()
+    plant = models.ForeignKey(Plant, on_delete=models.CASCADE)
```
Comment (halypa) [inline plants/models.py]: plant = models.ForeignKey(Plant, on_delete=models.CASCADE, related_name='pictures')

File: plants/models.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    # weather = models.ForeignKey(User, on_delete=models.CASCADE)
+    # location
+
+    def __str__(self):
+        return f"Profile owner: {self.user}"
+
+
+class CommonName(models.Model):
+    name = models.CharField(max_length=200)
+    plant = models.ForeignKey(Plant, on_delete=models.CASCADE)
```
Comment (halypa) [inline plants/models.py]: related_name

File: plants/models.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+    image = models.ImageField()
+    plant = models.ForeignKey(Plant, on_delete=models.CASCADE)
+    profile_plant = models.ForeignKey(ProfilePlant, on_delete=models.CASCADE)
+    created = models.DateTimeField(auto_now_add=True)
+    update = models.DateTimeField(auto_now=True, null=True, blank=True)
+
+
+DURATION = (("Annual", "Annual"), ("Biennial", "Biennial"), ("Perennial", "Perennial"))
+
+TOXICITY = (("None", "None"), ("Low", "Low"), ("Medium", "Medium"), ("High", "High"))
```
Comment (halypa) [inline plants/models.py]: https://django-model-utils.readthedocs.io/en/latest/utilities.html#choices

**Comment by wayzor9** [comment]
here comes the time for new pull request on this branch!

**Pull request closed without merging**